### DISCUSSION TITLE
Follow file system links

### DISCUSSION THREAD

**Discussion by mchester50** [filebrowser/filebrowser#1611] [Q&A]
Is there an option to allow Filebrowser to follow links on unix file systems? I have created links to attached disks but file browser will not see the links. For example if I create a link to a disk with the unix name of dev-disk-by-uuid-5CD9-57EC/ and I link that the name Photos. Filebrowser will not follow that link. When you try to follow the linked directory you get the message "This location can't be reached". The link I'm trying to follow was created with the unix ln command. i.e. 'ln /srv/dev-disk-by-uuid-5CD9-57EC Photos' which creates a directory Photos linked to /srv/dev-disk-by-uuid-5CD9-57EC

**Comment by vadbess** [comment]
i have the same problem

**Comment by fpetter** [comment]
Same problem with symbolic links.
